apollo-portal/src/main/java/com/ctrip/framework/apollo/portal/controller/PermissionController.java

```diff
@@ -415,11 +415,11 @@ public List<String> getCreateApplicationRoleUsers() {
   @GetMapping("/system/role/createApplication/{userId}")
   public JsonObject hasCreateApplicationPermission(@PathVariable String userId) {
     JsonObject rs = new JsonObject();
-    rs.addProperty("hasCreateApplicationPermission", userPermissionValidator.hasCreateApplicationPermission(userId));
+    rs.addProperty("hasCreateApplicationPermission", unifiedPermissionValidator.hasCreateApplicationPermission());
```

Comment:
The original implementation included `userId`; why was it removed in the update?